turbo.json

```diff
@@ -5,6 +5,7 @@
     "clean": {},
     "setup:test": {},
     "teardown": {},
-    "test-start": {}
+    "test-start": {},
```

Comment:
whats all this ah? I have already added these commands in turbo.json

Reply:
There was a dev command missing previously!

Reply:
true, but the latest main alr has these now, so these changes are not required

Reply:
ok removed